src/ai/defaultai.cc

```diff
@@ -269,6 +269,17 @@ void DefaultAI::think() {
 		}
 		return;
 	}
+	for (const Widelands::Game::PendingDiplomacyAction& pda : game().pending_diplomacy_actions()) {
```

Comment:
can we make this random with a low chance to make it less forseeable and fake some strategic decision for v1.1